src/chart.titleclick.js

```diff
@@ -36,6 +36,8 @@ var handleEvent = function(titleBlock, e) {
             // use e.native for backwards compatibility
             opts.onHover.call(me, e.native, me);
         }
+    } else if (type === 'mousemove') {
```

Comment:
To make it backwards-compatible if `opts.onLeave` is unset, what about this?

```suggestion
    } else if (type === 'mousemove' && opts.onLeave) {
```